tests/integration/components/custom-form/link-to-form-with-modal-test.js

```diff
@@ -0,0 +1,27 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'frontend-lmb/tests/helpers';
+import { render } from '@ember/test-helpers';
```

Comment:
remove tests unless they have meaningful content. These will likely fail because of missing expected parameters